golang/frontend/main.go

```diff
@@ -18,17 +18,19 @@ import (
 	"github.com/honeycombio/opentelemetry-exporter-go/honeycomb"
 )
 
-const (
-	nameServiceUrl    = "http://localhost:8000/name"
-	messageServiceUrl = "http://localhost:9000/message"
+var (
+	apiKey            = os.Getenv("HONEYCOMB_API_KEY")
+	dataset           = os.Getenv("HONEYCOMB_DATASET")
+	nameServiceUrl    = os.Getenv("NAME_ENDPOINT") + "/name"
```

Comment:
Do these have defaults set somewhere (a la the Python code)?

Reply:
Nnnnno. You're right; they should get some!

Reply:
Trying to update this has turned into a yak shave and I don't know why.